data/json/items/chemicals_and_resources.json

```diff
@@ -824,6 +824,22 @@
     "category" : "chems",
     "fun": -20
    },
+   {
+    "type" : "AMMO",
+    "id" : "nanomaterial",
+    "category" : "chems",
+    "price" : 0,
```

Comment:
Shouldn't the nanomaterial have some pricing? I don't personally know how much should they cost , but I asume that they are costly due to their spawnrate and the things yo can achieve with it.